components/win-users/tests/account.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate habitat_win_users;
+extern crate winapi;
+
+use std::env;
+
+use winapi::winnt::{SidTypeUser, SidTypeWellKnownGroup};
+
+use habitat_win_users::account::Account;
+
+#[test]
+fn real_account_returns_some() {
+  assert_eq!(Account::from_name("Administrator".to_string()).is_some(), true)
+}
+
+#[test]
+fn bogus_account_returns_none() {
+  assert_eq!(Account::from_name("bogus".to_string()).is_none(), true)
+}
+
+#[test]
+fn user_account_returns_user_type() {
+  let acct_type = Account::from_name("Administrator".to_string()).unwrap().account_type;
```

Comment:
given the cross platform nature of the code base and how much case matter elsewhere, might be worth including in a test or doc test just for clarification/edification of newcomers.